jscomp/test/bs_unwrap_test.ml

```diff
@@ -0,0 +1,49 @@
+external log1 :
+  (
+    [ `Pair of string * int
+    | `Int of int
```

Comment:
FWIW, `unbox`is the easier term to relate to for me, but my impression is that `unwrap` is the more established term, and is used with rust's `Option` type for example: https://doc.rust-lang.org/std/option/enum.Option.html#method.unwrap